src/librustdoc/html/render/mod.rs

```diff
@@ -1248,23 +1252,26 @@ fn notable_traits_decl(decl: &clean::FnDecl, cx: &Context<'_>) -> String {
     out.into_inner()
 }
 
+#[derive(Clone, Copy, Debug)]
+struct ImplRenderingParameters {
+    show_def_docs: bool,
+    is_on_foreign_type: bool,
+    show_default_items: bool,
+    /// It'll exclude methods if set to `false`.
```

Comment:
"Whether or not to show methods and associated constants"

Reply:
The associated constants are included in any case. I'll reword it though, your wording is better.